aiinfra-cluster/installation_scripts/install_cloud_ops_agent.sh

```diff
@@ -43,7 +136,42 @@ handle_debian() {
     }
 
     install_opsagent() {
-        curl -s https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.sh | bash -s -- --also-install
+        curl -s https://dl.google.com/cloudagents/add-google-cloud-ops-agent-repo.sh |
+            REPO_SUFFIX=20230214-1.1.1 bash -s -- --also-install
+    }
+
+    install_dcgm() {
+        echo 'nvidia-uvm' >>/etc/modules
+
+        cat >/etc/google-cloud-ops-agent/config.yaml <<EOF
+metrics:
+  receivers:
+    hostmetrics:
+      type: hostmetrics
+      collection_interval: 10s
+    nvml:
+      type: nvml
+      collection_interval: 10s
+    dcgm:
+      type: dcgm
+      collection_interval: 10s
+  service:
+    pipelines:
+      dcgm:
+        receivers:
+          - dcgm
+EOF
+
+        local distribution=$(. /etc/os-release;echo $ID$VERSION_ID | sed -e 's/\.//g')
+        echo "Installing DCGM for distribution '${distribution}'"
+        wget "https://developer.download.nvidia.com/compute/cuda/repos/${distribution}/x86_64/cuda-keyring_1.0-1_all.deb"
+        dpkg -i cuda-keyring_1.0-1_all.deb && rm cuda-keyring_1.0-1_all.deb
+        apt-get update
+        apt-get install -y datacenter-gpu-manager
```

Comment:
Do you need to run the post-install step afterwards?
https://docs.nvidia.com/datacenter/dcgm/latest/user-guide/getting-started.html#post-install

Reply:
Yup. I had the start command but forgot the enable. Updated.